english/sing-song/vlad.md

```diff
@@ -0,0 +1 @@
+[Thank God. Am so happy to be at the end.] (www.crazyvlad.com)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.